components/nft-sale-widget/layout/nft-sale/mint-specific-psyc-header.tsx

```diff
@@ -1,11 +1,14 @@
 import { Box, Flex, Text } from "@chakra-ui/react";
-import DecorationFrame from "../../commons/decoration-frame";
-import DiagonalRectangle from "../../commons/diagonal-rectangle";
+import DecorationFrame from "../../common/decoration-frame";
+import DiagonalRectangle from "../../common/diagonal-rectangle";
+import { useSaleWidget } from "@/providers/SaleWidgetContext";
+
+// type MintSpecificPsycHeaderProps = {
```

Comment:
why didn't you just delete this set of Props if it's no longer necessary?

Reply:
will do. was an oversight